### PR TITLE
Removed double-brace initialized collection

### PR DESCRIPTION
**Pull request by Miha-x64**
There's no need to create anonymous class extending TreeMap through double-brace initialization: it is a very bad practice which should be avoided, especially in example projects; also, there's no need to recreate this map with Activity instance (e. g. due to screen rotarion) so it made static.

### CODE REVIEW

**Comment by zaki50** [comment]
I agree with that we don't need to recreate the map on rotation.
However, keeping the map as static prevents the map to be GCed.

How about removing the map and modify `setupButtons()` like:
```
    private void setupButtons() {
        setupButton("ListView", ListViewExampleActivity.class);
        setupButton("RecyclerView", RecyclerViewExampleActivity.class);
    }

    private void setupButton(String label, final Class<? extends Activity> activityClass) {
        Button button = new Button(this);
        button.setText(label);
        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(activityClass);
            }
        });
        container.addView(button);
    }
```

**Comment by zaki50** [comment]
@Miha-x64 Could you change the target branch of this PR to `releases`?

File: example/src/main/java/io/realm/examples/adapters/MainActivity.java [hunk 1]
```diff
@@ -32,11 +32,15 @@
 
 public class MainActivity extends AppCompatActivity {
 
+    private static final Map<String, Class<? extends Activity>> buttons;
+    static {
+        Map<String, Class<? extends Activity>> map = new TreeMap<String, Class<? extends Activity>>();
+        map.put("ListView", ListViewExampleActivity.class);
+        map.put("RecyclerView", RecyclerViewExampleActivity.class);
+        buttons = Collections.unmodifiableMap(map);
```
Comment (emanuelez) [inline example/src/main/java/io/realm/examples/adapters/MainActivity.java]: Missing the importing of `java.util.Collections`?

**Comment by zaki50** [comment]
@Miha-x64 And could you sign our CLA? https://github.com/realm/realm-android-adapters/blob/master/CONTRIBUTING.md#contributing-enhancements

**Comment by Miha-x64** [comment]
Fixed as suggested. There's no collection at all.
Unfortunately, I can't sign CLA because I'm located in the middle of nowhere and have no appropriate mailing address.

**Comment by bmeike** [comment]
Closed for lack of CLA.  Issue now being tracked here: https://github.com/realm/realm-android-adapters/pull/82

**Comment by zaki50** [comment]
We can accept CLA without mailing address.
I'll update the CLA form.

**Pull request closed without merging**